### ISSUE TITLE
babel-node error when passing arguments

### ISSUE THREAD

**Issue by u9520107** [babel/babel#5161]
### Input Code

When trying to run index.js with arguments, babel-node will error out.

```bash
babel-node index.js test
```

### Babel Configuration (.babelrc, package.json, cli command)

Config is not invoked as babel-node fails to identify the correct path to index.js
```js
{
  "your": { "config": "here" }
}
```

### Expected Behavior

babel-node runs index.js with "test" in the argument list

### Current Behavior

The following Error is shown:
```
module.js:472
    throw err;
    ^

Error: Cannot find module '/Users/jack.huang/github/u9520107/ringcentral-js-integration-commons/test'
    at Function.Module._reso lveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Function.Module.runMain (module.js:605:10)
    at /Users/jack.huang/.nvm/versions/node/v7.2.0/lib/node_modules/babel-cli/lib/_babel-node.js:155:24
    at Object.<anonymous> (/Users/jack.huang/.nvm/versions/node/v7.2.0/lib/node_modules/babel-cli/lib/_babel-node.js:156:7)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

### Context

This affects our project build script as we write gulpfiles with es6. We currently do things like the following, and would run into this bug.

```bash
./node_modules/babel-cli/bin/babel-node.js ./node_modules/gulp/bin/gulp.js test
```

### Your Environment

| software         | version
| ---------------- | -------
| Babel            |  6.22.1
| node             | 7.2.0
| npm              |  3.10.9  (also used yarn 0.19.1)
| Operating System |  maxos (also tested on linux, but not sure which distro)

**Comment by joscha** [comment]
we are seeing the same problem where previously this command:

```
babel-node ~/src/main.js --dir /bla/static/js --files ./base/preconditions.js ./base/serialization.js ./services/print/dummy_print_service.js ./services/print/print_proto.js ./services/print/print_service.js --entry-points services/print/dummy_print_service.js
```

would work, but now errors with:
```
Error: Cannot find module '/Users/joscha/dev/canva/cfe/services/print/dummy_print_service.js'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Function.Module.runMain (module.js:590:10)
    at /Users/joscha/dev/canva/tools/gendeps/node_modules/babel-cli/lib/_babel-node.js:155:24
    at Object.<anonymous> (/Users/joscha/dev/canva/tools/gendeps/node_modules/babel-cli/lib/_babel-node.js:156:7)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

**Comment by loganfsmyth** [comment]
Fixed in `babel-cli@6.22.2`, sorry about that and thanks for the bug report.

**Comment by u9520107** [comment]
@loganfsmyth Thanks for the quick fix!
